### DISCUSSION TITLE
public server

### DISCUSSION THREAD

**Discussion by farshad-fallah** [BearWare/TeamTalk5#2148] [Q&A]
hi hello . . could you plese guide me how i an create public server team talk and shown in the list of servers in the application 
  thanks before

**Comment by bear101** [comment]
Checkout https://bearware.dk/teamtalk/v5.13.1/docs/userinterface.html#connectdlg and locate the "Publish Publicly" button.

**Comment by farshad-fallah** [comment]
thanks but i find only about client 
there is not about config   public server or unofficial servers
could you plese guide me

**Comment by farshad-fallah** [comment]
sorry
